ts/src/v8-types.ts

```diff
@@ -14,6 +14,8 @@
  * limitations under the License.
  */
 
+const heapProfiler = require('bindings')('sampling_heap_profiler');
```

Comment:
Is this expected to be hear? It's not obvious why this belongs to this file (and why only for the heap profiler, not time).

Reply:
Thanks. That was an error.